example-jobs/pipeline_job.groovy

```diff
@@ -0,0 +1,48 @@
+import jenkins.automation.builders.PipelineJobBuilder
+
+def script = """
```

Comment:
This is cleaner when defined as stripping the top line, and not including the bottom line:

```
dec script ="""\
...
}"""
```

Reply:
To elaborate, here's what I had in my test script

```
def script = """\
node {
  def buildSteps = [:]

  buildSteps['Build'] = {
    sh 'env | sort'
    echo 'Build step'
    sh 'uptime'
    echo '\$class'
  }

  buildSteps['Test'] = {
    sh 'echo teeeessssttttt'
    echo 'Test step'
    sh 'uptime'
  }

  parallel buildSteps
}"""
```

Which results in

![image](https://user-images.githubusercontent.com/6773712/31092010-fef7ab52-a77a-11e7-8b1c-ae6467b95821.png)


If I have
```
def script = """
node {
  def buildSteps = [:]

  buildSteps['Build'] = {
    sh 'env | sort'
    echo 'Build step'
    sh 'uptime'
    echo '\$class'
  }

  buildSteps['Test'] = {
    sh 'echo teeeessssttttt'
    echo 'Test step'
    sh 'uptime'
  }

  parallel buildSteps
}
"""
```

Then I end up with
![image](https://user-images.githubusercontent.com/6773712/31092067-33c90b5a-a77b-11e7-9a86-a7fc772311d9.png)

Notice the extra lines at the beginning and end.

Reply:
Also notice escape slash before $class.

Reply:
Yes, true, but all things considered, the functionality is the same. Slash may see a little more magical to non-groovy people, while stripIndent() is more expressive.  I am not inclined to add another commit just for that, if you don't object.